includes/admin/class-sensei-learner-management.php

```diff
@@ -166,6 +166,8 @@ public function enqueue_scripts() {
 			)
 		);
 
+		wp_enqueue_script( 'jquery-ui-tooltip' );
```

Comment:
Could this just be a dependency of the line where `learners-general.js` is being enqueued?

Reply:
Yup! Good call. ec0e0aa60990d94835c21c1c90817a052545a2a5